### PR TITLE
Add Order - View customer details : Check if nav-bar exist to avoid console errors

### LINKED ISSUE #20651: [BO][CREATE AN ORDER] Errors in console
A clear and concise description of what went wrong.

In BO, Open the console. Add a new order and see the errors in console

Explain what you expected to happen instead.

#### Steps to Reproduce

1. In BO, open your console and Go to Order>Create Order> Create a new customer or select existing customer and click on "details"
==>See the errors in console 

**Screenshots**

Capture d’écran 2020-08-20 à 10 54 58

select exisiting customer, click on "details"

Capture d’écran 2020-08-20 à 10 55 19

#### Additional information

* PrestaShop version: 1.7.7.x
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Check if nav-bar exists when displaying customer details in Create order page. This will avoid to call functions related to it
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #20651
| How to test?  | First open your browser console<br>In BO, Go to Order>Create Order<br>Create a new customer or select existing customer<br>Click on "details"<br>You wont see the JS errors described<br>Capture d’écran 2020-08-20 à 10 54 58

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20760)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Comment by ngodefroy** [comment]
Hello,

I was able to reproduce parti of the issue with the onboarding html error.
See attached screenshot:

Capture d’écran 2020-09-01 à 14 57 26

File: admin-dev/themes/default/template/header.tpl [hunk 1]
```diff
@@ -400,7 +400,9 @@
       </div>
 {/if}
 
+{if $display_header}
```
Comment (PierreRambaud) [inline admin-dev/themes/default/template/header.tpl]: If I'm right, you're already in the `if $display_header` which begins on line 110 :thinking:

File: admin-dev/themes/new-theme/template/layout.tpl [hunk 2]
```diff
@@ -92,7 +92,9 @@
 
       <div class="{if $display_header}content-div{/if} {if !isset($page_header_toolbar)}-notoolbar{/if} {if $current_tab_level == 3}with-tabs{/if}">
 
-        {hook h='displayAdminAfterHeader'}
+        {if $display_header}
+          {hook h='displayAdminAfterHeader'}
+        {/if}
 
         {if $display_header}
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/template/layout.tpl]:
```suggestion
        {if $display_header}
          {hook h='displayAdminAfterHeader'}

```

Comment (PierreRambaud) [reply admin-dev/themes/new-theme/template/layout.tpl]: You maybe need to do the same on the default theme

File: admin-dev/themes/new-theme/template/layout.tpl [hunk 3]
```diff
@@ -92,9 +92,9 @@
 
       <div class="{if $display_header}content-div{/if} {if !isset($page_header_toolbar)}-notoolbar{/if} {if $current_tab_level == 3}with-tabs{/if}">
 
-        {hook h='displayAdminAfterHeader'}
-
         {if $display_header}
+          {hook h='displayAdminAfterHeader'}
+
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/template/layout.tpl]: Isn't a BC ?

Comment (Progi1984) [reply admin-dev/themes/new-theme/template/layout.tpl]: Ping @PrestaShop/prestashop-core-developers ?

**Review by Progi1984 — COMMENTED** [review]
I think it's a BC

File: admin-dev/themes/new-theme/template/layout.tpl (hunk 3 above)
Comment (sowbiba) [reply admin-dev/themes/new-theme/template/layout.tpl]: Seems logic to me to trigger displayAdminAfterHeader only if header is displayed

Do I just have to notice it in the PR description ?

Comment (Progi1984) [reply admin-dev/themes/new-theme/template/layout.tpl]: Yes

**Comment by sarahdib** [comment]
Hey @sowbiba this error is still displayed but the second one isn't displayed anymore. Is it normal ?
Capture d’écran 2020-09-11 à 15 59 47

**Review by Progi1984 — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by jolelievre** [comment]
@sowbiba I believe the test description might need some updates The core fix will only fix the `PerfectScrollbar` error right?
And the `OnBoarding` error will be managed in the module?

**Review by PierreRambaud — APPROVED** [review]

**Comment by ngodefroy** [comment]
TEST OK with new or old customer.
Capture d’écran 2020-09-25 à 13 41 07

Thank you @sowbiba

**Comment by NeOMakinG** [comment]
Thanks @sowbiba

**Pull request merged**